src/ConnectionState.hpp

```diff
@@ -55,6 +58,6 @@ class ConnectionState
 
   PortType _requiredPort;
 
-  Node* _lastHoveredNode = nullptr;
+  Node* _lastHoveredNode{nullptr};
```

Comment:
In this case not very much, but List initialization should be preferred as it does not allow narrowing.